### PR TITLE
Add normalize.css

### PR DESCRIPTION
**Pull request by VilhelmNielsen**

### CODE REVIEW

File: styles/normalize.css [hunk 1]
```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (augustocravosilva) [inline styles/normalize.css]: Maybe add
```
/***** Normalize.css *****/
```
on top

Comment (VilhelmNielsen) [reply styles/normalize.css]: Good call, ammended 👍

File: style.css [hunk 2]
```diff
@@ -1,4 +1,187 @@
...
+  padding: 0.35em 0.75em 0.625em;
+}
+
+legend {
+  box-sizing: border-box;
+  color: inherit;
+  display: table;
+  max-width: 100%;
+  padding: 0;
+  /* 3 */
```
Comment (augustocravosilva) [inline style.css]: missed this one

Comment (VilhelmNielsen) [reply style.css]: 😅

Comment (VilhelmNielsen) [reply style.css]: @augustocravosilva amended 8da76dd

**Pull request merged**